vscode/src/workspace.ts

```diff
@@ -14,6 +14,13 @@ import {
 } from "./common";
 import { WorkspaceChannel } from "./workspaceChannel";
 
+const WATCHED_FILES = [
+  "Gemfile.lock",
+  "gems.locked",
+  ".rubocop.yml",
```

Comment:
It looks like we were previously matching in `.rubocop.yml` in any location, but now only in the root?